### ISSUE TITLE
Simplify test profiles in CI test

### ISSUE THREAD

**Issue by LilyAnderssonLee** [nf-core/taxprofiler#374]
### Description of feature

As pointed out by @maxulysse in #362, we should simplify the test profiles since the test is already included in the CI tests.
Therefore should we just keep the following test profiles from ci.yml?
 
- "test"
- "test_malt"
- "test_motus"
- "test_prinseqplusplus"

**Comment by jfy133** [comment]
Yeah this definitely would be helpful to `include` the base test config and modify off of that, just came across a case where I was modifying `test` but was actually running `test_noprofiling`... however the addition I made to `test` would apply to all the data :sweat_smile: - thus makes sense to apply it 'everywhere'

**Comment by jfy133** [comment]
I've refined the CI tests while adding each nf-test profile, so this has been done ovcer time
